pkg/apis/core/register.go

```diff
@@ -98,5 +98,15 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&ConfigMapList{},
 	)
 
+	// TODO: should this depend on the feature gate?
+	scheme.AddKnownTypes(SchemeGroupVersion,
```

Comment:
Where is support for the new types in the core API meant to be conditional? Also here in the schema or only in the registration of the storage?

Reply:
It's enough to make registration of the storage depend on the feature gate (https://kubernetes.slack.com/archives/C0EG7JC6T/p1664286455977939?thread_ts=1664283458.731629&cid=C0EG7JC6T).